README.md

```diff
@@ -1,2 +1,42 @@
-# coin
-contextual outlier interpretation
+# COIN
+## Contextual Outlier INterpretation
+In this package, we implement the Contextual Outlier INterpretation (COIN) method that aims to explain the 
+abnormality of existing outliers spotted by detectors.
```

Comment:
In this package, we implement the Contextual Outlier INterpretation (COIN) method that aims to explain the 
abnormality of outliers spotted by an outlier detector.